packages/streamr-logger/include/streamr-logger/Logger.hpp

```diff
@@ -1,18 +1,152 @@
-#ifndef STREAMER_UTILS_LOGGER_LOGGER_HPP
-#define STREAMER_UTILS_LOGGER_LOGGER_HPP
+#ifndef STREAMER_UTILS_LOGGER_LOGGER_H
+#define STREAMER_UTILS_LOGGER_LOGGER_H
 
-#include <iostream>
+#include <folly/logging/LogConfig.h>
+#include <folly/logging/LogLevel.h>
+#include <folly/logging/LogMessage.h>
+#include <folly/logging/LogWriter.h>
+#include <folly/logging/LoggerDB.h>
+#include <folly/logging/xlog.h>
+#include "StreamrWriterFactory.hpp"
+#include "StreamrHandlerFactory.hpp"
+
+namespace streamr {
+namespace logger {
+
+namespace {
```

Comment:
Do not use unnamed namespaces in header files, it is potentially dangerous, see discussion: https://stackoverflow.com/questions/357564/are-there-any-uses-for-unnamed-namespaces-in-header-files